### DISCUSSION TITLE
YouTube video in English downloads in another languge 8.2.2.8a1 (beta)?

### DISCUSSION THREAD

**Discussion by exquirentibus** [aclap-dev/video-downloadhelper#423] [Help and Q&A]
### Your question

I'm running in Firefox 125.0.2 on Windows 10 with 8.2.2.8a1 (beta) using the VdhCoApp 2.0.19.  I am licensed for the plug-in.

YouTube videos have been working perfectly.  I downloaded several today.  My focus is technical videos to view off-line.  I'm not doing anything to block the Ads which play until I skip them when asked.

This video plays on YouTube in English. 

https://youtu.be/2fKIaalk4_w

However, the downloaded version of Every download plays a few seconds in English and then switches to possibly Spanish?  It may be that the Spanish is behing forced on Top of the English as I think I can hear some English at low volume beneath the Spanish?

### Addon & CoApp version and details

version: 8.2.2.8a1
channel: beta
build date: 2024-03-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 7b801cff14e0466e
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Win10Test",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I suggest you read #389 carefully.  Your case looks like a similar issue.  Be grateful there's only 3 languages on your video.  There were something like 20 languages on the other one.  The short answer is that VDH seems incapable of getting the right language at the moment.  But you can do it yourself without using VDH.  All it takes is persistence & determination.
